cfn-resources/cluster/mongodb-atlas-cluster.json

```diff
@@ -275,6 +275,21 @@
                 }
             },
             "additionalProperties": false
+        },
+        "tag": {
```

Comment:
My opinion is that tag is the correct name, because the property where this definition is referenced is a list and is already called tags, so is a property called tags, that is a list of tag